meta1v2/meta1_backend_references.c

```diff
@@ -110,6 +110,17 @@ __destroy_container(struct sqlx_sqlite3_s *sq3, struct oio_url_s *url,
 	return err;
 }
 
+static void
+__notify_account(struct meta1_backend_s *m1, struct oio_url_s *url)
+{
+	if (!m1->notifier)
+		return;
+
+	GString *notif = oio_event__create("storage.account.new", url);
+	g_string_append_static (notif, ",\"data\":[]}");
+	oio_events_queue__send(m1->notifier, g_string_free (notif, FALSE));
+}
```

Comment:
I think we would better reuse the `account.services` event type, and send an empty list of services. Unfortunately we never managed these events, but they are still send when the list of services linked to a reference changes.
The code is in https://github.com/open-io/oio-sds/blob/master/meta1v2/meta1_backend_services.c#L1217, you will have to refactor a little bit.